modules/core/src/main/java/org/apache/ignite/internal/client/thin/ClientOperation.java

```diff
@@ -280,7 +280,10 @@ public enum ClientOperation {
     OP_SET_ITERATOR_START(9022),
 
     /** IgniteSet.iterator page. */
-    OP_SET_ITERATOR_GET_PAGE(9023);
+    OP_SET_ITERATOR_GET_PAGE(9023),
+
+    /** Get service topolog. */
+    SERVICE_GET_TOPOLOGY(9024);
```

Comment:
Please move up to other service-related operations, and use `7003` code (make sure to update the IEP as well)

Reply:
Fixed